tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -217,14 +217,38 @@ def __init__(self, prefix):
         self.total = None
         self.current = None
 
+    def _current_partition_count(self, ctx):
+
+        rpk_cnt = len(list(ctx.rpk().describe_topic(self.topic,
```

Comment:
It's unclear why do you use `describe_topic` along with `get_partitions`, the latter should be enough. the only reason I see is that you decided to test that `describe_topic` returns the same to maintain `rpk.describe_topic` coverage but in that case it makes sense to log or throw error when it's false.

Also the `.count()` logic looks a bit complicated, suggesting to go with the following structure:

```
partition_counts = set()
for n in ctx.redpanda.started_nodes():
    cnt = 0
    partitions = ctx.admin().get_partitions(node=n)
    for p in partitions:
        ...
    log("node {n} knows about {cnt} partitions")
    # a comment on why we ignore the zero case
    if cnt != 0:
        partition_counts.add(cnt)

if len(partition_counts)==0:
    log("topic isn't found")
    return None

if len(partition_counts)>1:
    log("nodes don't agree on the partition count")

partition_count = next(iter(partition_counts))
rpk_count = len(list(ctx.rpk().describe_topic(..)))

if partition_count != rpk_count:
     log("rpk is broken")
     raise Exception(...)

return partition_count
```

Reply:
When i think of it now the solution based on admin API is not going to work as nodes may not host the same number of partitions for the same topic

Reply:
we need to change this completely